src/main/scala/net/snowflake/spark/snowflake/FilterPushdown.scala

```diff
@@ -57,56 +57,6 @@ private[snowflake] object FilterPushdown {
     keepNameCase: Boolean
   ): Option[SnowflakeSQLStatement] = {
 
```

Comment:
Explain to reviewers:
These two functions is moved out of buildFilterStatement() so that I can add some unit test for them.